azure/packages/azure-client/src/AzureUrlResolver.ts

```diff
@@ -93,6 +93,12 @@ function decodeAzureUrl(urlString: string): {
     };
 }
 
+/**
+ * Creates a request for creation of a new Fluid Container.
```

Comment:
This function is kind of inherently confusing unfortunately - maybe something like this would help clarify a bit?
```suggestion
 * Creates a request object that can be passed to a container.attach() call to request creation of a new Fluid Container on the Azure service.
```

Reply:
Pushed a slightly modified version of that, which links to the `attach` method. Thanks!